compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -9258,7 +9420,11 @@ TR::Node *ddivSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    // In IEEE FP Arithmetic, d / 1.0 is d
    // Pretend to be an int before doing the comparison
    //
-   BINARY_IDENTITY_OP(LongInt, DOUBLE_ONE)
+   auto binOpSimplifier = getLongBinaryOpSimplifier(s);
+   auto identity = binOpSimplifier.tryAndSimplifyIdentityOp(node, (int64_t) DOUBLE_ONE);
```

Comment:
Similar comment as above, for `ddivSimplifier`, I think this one should be updated to `getDoubleBitsBinaryOpSimplifier` instead of using `getLongBinaryOpSimplifier`.